### PR TITLE
Change message in bulk edit location

### PR DESCRIPTION
**Pull request by szok**
Adding additional error messages when bulk editing location

### CODE REVIEW

File: src/ralph_assets/tests/unit/tests_other.py [hunk 1]
```diff
@@ -520,3 +521,16 @@ def test_include_blank(self):
...
+class TestBulkEditLocationMessage(TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_get_non_blade_message(self):
+        chassis_bulk_edit = ChassisBulkEdit()
+        assets = [DCAssetFactory(sn='test')]
+        msg_sn, msg_barcode = chassis_bulk_edit._get_non_blade_message(assets)
+        self.assertEqual(bool(msg_sn), True)
```
Comment (ar4s) [inline src/ralph_assets/tests/unit/tests_other.py]: IMO ``assertTrue`` is better for this case

File: src/ralph_assets/tests/unit/tests_other.py [hunk 2]
```diff
@@ -520,3 +521,16 @@ def test_include_blank(self):
...
+        self.assertEqual(bool(msg_barcode), False)
```
Comment (ar4s) [inline src/ralph_assets/tests/unit/tests_other.py]: similary - ``assertFalse``

**Comment by ar4s** [comment]
What about other cases? Like (``msg_sn=False``, ``msg_barcode=False``), (``msg_sn=True``, ``msg_barcode=True``), (``msg_sn=False``, ``msg_barcode=True``).

File: src/ralph_assets/tests/unit/tests_other.py [hunk 3]
```diff
@@ -520,3 +521,33 @@ def test_include_blank(self):
...
         BLANK_IDX = 0
         self.assertNotIn(found[BLANK_IDX], models_assets.AssetStatus())
+
+
+class TestBulkEditLocationMessage(TestCase):
+
+    def setUp(self):
+        pass
+
+    def test_get_non_blade_message(self):
```
Comment (xliiv) [inline src/ralph_assets/tests/unit/tests_other.py]: I'd split this to 3 tests:
- test_get_non_blade_with_sn_message
- test_get_non_blade_with_barcode_message
- test_get_non_blade_with_sn_and_barcode_message

**Comment by xliiv** [comment]
:ok:

**Comment by ar4s** [comment]
:ok:

**Pull request merged**